README.md

```diff
@@ -627,9 +627,7 @@ p.then((val) => console.log("Promise Resolved", val),
 Returning a value from `then` callbacks will pass the value to the next `then` callback.
 
 ```javascript
-var hello = new Promise(function(resolve, reject) {  
-    resolve("Hello");
-});
+var hello = new Promise((resolve, reject) => resolve("Hello"));
```

Comment:
Hi @cesarcosta99, you are returning resolve("Hello") here, you should instead write it as:

`new Promise((resolve, reject) => { resolve("Hello") });`

Similarly at line 641 and 643